lib/OpenQA/WebAPI.pm

```diff
@@ -553,5 +571,110 @@ sub run {
     Mojolicious::Commands->start_app('OpenQA::WebAPI');
 }
 
+sub get_pod_from_controllers {
```

Comment:
I'd suggest to move this to it's own package along with _get_pod_text so that webAPI has a single responsability

Reply:
So ... OpenQA::WebAPI::Documentation?